control/joy_controller/config/joy_controller.param.yaml

```diff
@@ -1,5 +1,6 @@
 /**:
   ros__parameters:
+    joy_type: "P65"
```

Comment:
It seems like default parameter is set to `DS4` when it is launch from a [launch file](https://github.com/autowarefoundation/autoware.universe/blob/7241c83310e7d806ba1150274204d3a08859937d/control/joy_controller/launch/joy_controller.launch.xml#L2).
Maybe it's better to make the default value "DS4" to be consistent with the default value in launch file.

Also, since we decided to not have default values defined in places other than schema file, we might want to consider removing the default value set in launch file as well.

Reply:
I think DS4 would be better since we usually run the node from the launch file.